pypopminer/src/miner.cpp

```diff
@@ -25,9 +25,6 @@ struct Payloads {
     list listctx;
```

Comment:
you should add `std::vector<VbkBlock> context;` in this struct

and serialize each block and then `.append` in this list